### PR TITLE
Add doc on how to contribute

### PR DESCRIPTION
**Pull request by stevenjmesser**
This will help non-developers on the team contribute to the playbook.

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+This document is a guide on contributing to the team docs. It is a live guide and may not be complete, so you can raise problems or queries with a delivery manager or product manager.
+
+## Who can contribute
+
+Contributions must come from people on the GOV.UK Design System team. They are the users of these docs and the subject matter experts.
+
+## What to contribute
+
+This playbook is for you. It's a reference, a set of docs to turn to when you're unsure about anything, so contribute anything that will help you in your day job.
```
Comment (davidc-gds) [inline contributing.md:15]:
```suggestion
This playbook is for you. It's a reference, a set of docs to turn to when you're unsure about something, so contribute anything that will help you in your day job. 
```
A pedantic change, to note that we won't find answers on _anything_ but we will for some things. 😛

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+## Who can contribute
+
+Contributions must come from people on the GOV.UK Design System team. They are the users of these docs and the subject matter experts.
+
+## What to contribute
+
+This playbook is for you. It's a reference, a set of docs to turn to when you're unsure about anything, so contribute anything that will help you in your day job. 
+
+Step-by-step guides on how to do things and our processes are good. Models, principles, ways of thinking and examples are great. It helps us learn what we do, how and why.
```
Comment (davidc-gds) [inline contributing.md:17]: Small clarity change:
```suggestion
Step-by-step guides on how to do things and our processes are good. Models, principles, ways of thinking and examples are great. It helps us learn what we do, and how and why we do it.
```

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## How to contribute
+
+How you contribute will depend on your familiarity with working on software, but contributing to the playbook is a safe way to build your skills.
```
Comment (davidc-gds) [inline contributing.md:21]:
```suggestion
How you contribute will depend on your familiarity with working on GitHub, but contributing to the playbook is a safe way to build your skills. 
```

File: contributing.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+A new tab will open which shows a preview of the team docs. You can now create and edit `.html.md.erb` files to add to and edit the docs. Your changes will save automatically but you will need to [commit your changes](https://docs.github.com/en/codespaces/developing-in-codespaces/using-source-control-in-your-codespace#committing-your-changes) to save them for good. 
+
+After you've made all the changes and additions you intended to make, [raise a pull request](https://docs.github.com/en/codespaces/developing-in-codespaces/using-source-control-in-your-codespace#raising-a-pull-request).
+
+Ask one of our friendly developers for help if you're doing this for the first time. Ask them if they would be happy to help in future too!
+
+### 4. Using your machine
+
+If you choose this option, you probably know how to do it. Create a branch, commit changes and raise a pull request. Your pull request will be reviewed and approved when ready. You can merge your PR once approved.
```
Comment (davidc-gds) [inline contributing.md:55]:
```suggestion
If you choose this option, you probably know how to do it. Create a branch, commit changes and raise a pull request. Your pull request will be reviewed and approved when ready. You can merge your pull request once approved.
```

File: contributing.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+### 5. Create an issue
+
+You can also create an issue to suggest additions and changes without working directly on the files. Add a new issue on the [Issues](https://github.com/alphagov/design-system-team-docs/issues) to suggest documentation for the playbook.
```
Comment (davidc-gds) [inline contributing.md:59]:
```suggestion
You can also create an issue to suggest additions and changes without working directly on the files. Add a new issue via the [Issues view](https://github.com/alphagov/design-system-team-docs/issues) to suggest documentation for the playbook.
```

File: contributing.md (hunk 1 above)
Comment (stevenjmesser) [reply contributing.md:15]: Hahaha, not pedantic at all! Good spot.

**Pull request merged**